README.md

```diff
@@ -80,6 +82,30 @@
 
 
 
+### Gifs - [Raytracing With Optix AI Denoising [Albedo and Normals]](https://www.youtube.com/watch?v=VFad-Y-dSxQ&feature=youtu.be)
+
+
+
+![](https://github.com/AnastaZIuk/IrrScreenShots/blob/master/denoising/den1.gif?raw=true)
+
+
+
+![](https://github.com/AnastaZIuk/IrrScreenShots/blob/master/denoising/den2.gif?raw=true)
+
+
+
+![](https://github.com/AnastaZIuk/IrrScreenShots/blob/master/denoising/den3.gif?raw=true)
+
+
+
+![](https://github.com/AnastaZIuk/IrrScreenShots/blob/master/denoising/den4.gif?raw=true)
+
+
+
+![](https://github.com/AnastaZIuk/IrrScreenShots/blob/master/denoising/den7.gif?raw=true)
```

Comment:
done use http links, put the images in ./site_media ( https://github.com/buildaworldnet/IrrlichtBAW/tree/master/site_media ) instead and then use local links/paths